### DISCUSSION TITLE
Release notes for teslausb-20231012-bullseye?

### DISCUSSION THREAD

**Discussion by smcasagrande** [marcone/teslausb#789] [General]
Are there (feature related) release notes for [teslausb-20231012-bullseye](https://github.com/marcone/teslausb/releases/tag/v4)? Not sure if it's useful to update from [teslausb-20211106.2](https://github.com/marcone/teslausb/releases/tag/v3.2).

I'm running on a Pi Zero W, 256GB flash, not using it for Music.

Thanks!

**Comment by marcone** [comment]
The features are in the TeslaUSB scripts that are downloaded at install time, not in the prebuilt image. If you want to update to the latest version of TeslaUSB, simply run [`setup-teslausb upgrade`](https://github.com/marcone/teslausb/wiki#general-tips). There is no reason to flash this image if you have an existing working install.

**Comment by smcasagrande** [comment]
OK, I guess I'm missing where the release notes are for the TeslaUSB scripts "current version" that are included if I was to run "setup-teslausb upgrade".  The last version I installed was the prebuilt (buster) image around May 2023, so where would I find any changes to the scripts since that timeframe?  This would help me understand what has changed since that time, as well as if it would potentially be useful to upgrade, or just sit on the current version of the scripts that I have installed.

Thanks for all you do with this project, marcone!

**Reply by marcone** [reply]
I don't maintain release notes because there aren't really "releases" as such. If you want to see what changed, you'd have to look at the [list of commits](https://github.com/marcone/teslausb/commits/main-dev).

**Reply by smcasagrande** [reply]
That works - thanks!

**Comment by matterbyte** [comment]
new to this project - would you advise against using the bullseye pre-release image to start with?

**Reply by marcone** [reply]
If you're new to TeslaUSB and/or linux, I would stick to the older image for now.
TeslaUSB functionality will be exactly the same.

**Reply by matterbyte** [reply]
will do, thanks
